.github/workflows/publish.yml

```diff
@@ -0,0 +1,91 @@
+# This is a GitHub workflow defining a set of jobs with a set of steps.
+# ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+#
+name: Publish
+
+# Trigger the workflow's on pushed tags or commits to main/master branch.
+on:
+  push:
+    branches: [main, master]
+    tags:
+
+defaults:
+  run:
+    # Declare bash be used by default in this workflow's "run" steps.
+    #
+    # NOTE: bash will by default run with:
+    #   --noprofile: Ignore ~/.profile etc.
+    #   --norc:      Ignore ~/.bashrc etc.
+    #   -e:          Exit directly on errors
+    #   -o pipefail: Don't mask errors from a command piped into another command
+    shell: bash
+
+jobs:
+
+  # Builds and pushes docker images to DockerHub and package the Helm chart and
+  # pushes it to jupyterhub/helm-chart@gh-pages where index.yaml represents the
+  # JupyterHub organization Helm chart repository.
+  #
+  # ref: https://github.com/jupyterhub/helm-chart
+  # ref: https://hub.docker.com/orgs/jupyterhub
+  #
+  Publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # chartpress requires the full history
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install chart publishing dependencies (chartpress, helm)
+        run: |
+          . ./ci/common
+          setup_helm
+          pip install --no-cache-dir chartpress
```

Comment:
We need a oneliner to install helm here since setup_helm is removed

Reply:
I added the function back. The only reason I removed it was because I thought "the k3s action sets it all up".